lib/dish.rb

```diff
@@ -0,0 +1,10 @@
+class Dish
+
+  attr_accessor :quantity
+  attr_reader :args
+
```

Comment:
public attr_accessors break encapsulation -a void them wherever possible.

What is this class actually doing? What is its responsibility?